scripts/test.sh

```diff
@@ -4,9 +4,10 @@
 set -e
 
 if [ "$CI" = "true" ]
-  then jest --projects packages/* --runInBand
-  else jest --projects packages/*
+  then jest --runInBand
```

Comment:
I wonder if it's still useful to have --runInBand (something to check)

Reply:
Indeed it's mostly for debugging (see [docs](https://facebook.github.io/jest/docs/en/cli.html#runinband)). I think we can remove it. Thanks!

Reply:
sometimes it's also faster on CI; but something to check out